asserts/device_asserts_test.go

```diff
@@ -121,6 +123,21 @@ func (mods *modelSuite) TestDecodeStoreIsOptional(c *C) {
 	c.Check(model.Store(), Equals, "")
 }
 
+func (mods *modelSuite) TestDecodeBaseIsOptional(c *C) {
+	withTimestamp := strings.Replace(modelExample, "TSLINE", mods.tsLine, 1)
+	encoded := strings.Replace(withTimestamp, "base: core18\n", "base: \n", 1)
+	a, err := asserts.Decode([]byte(encoded))
+	c.Assert(err, IsNil)
+	model := a.(*asserts.Model)
+	c.Check(model.Base(), Equals, "")
+
+	encoded = strings.Replace(withTimestamp, "base: core18\n", "", 1)
+	a, err = asserts.Decode([]byte(encoded))
+	c.Assert(err, IsNil)
+	model = a.(*asserts.Model)
+	c.Check(model.Base(), Equals, "")
```

Comment:
Should this return "core" for simplicity?

Reply:
all of asserts so far doesn't know about "core" the snap name,  I wouldn't add it here even now